.github/workflows/main-linux.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           -Pcode-coverage
       - name: SonarCloud Report
         #jdk8 is faster to run UT/IT
-        if: ${{ success() &&  matrix.java == '8'}}
+        if: ${{ success() &&  matrix.java == '11'}}
```

Comment:
emmm...how about remove or modify this comment?

Reply:
Hi, @HTHou . I agree with you. It's done.